dashboard/config/locales/data.en.yml

```diff
@@ -318,16 +318,17 @@ en:
-        'studio_iceage_8': "Can you add a block to score a point and a block to play a sound when Sid runs into Manny, and then move him with the arrows until you score?"
-        'studio_iceage_9': "Set the background for this ice age and help Sid get to Granny faster by changing its speed. Then move him with the arrow keys to get to Sid and alert everyone else."
-        'studio_iceage_10': "Create your own game. When you're done, click Finish to let friends try your game on their phones."
+        'studio_iceage_hello1': "Sid wants to gather everyone for a new adventure. Can you help him say \"hello\" by dragging the `say` block out into the work area and typing the word \"hello\" inside?"
+        'studio_iceage_hello2': "Sid (actor 1) found Manny (actor 2) and wants to invite him on the adventure.  Help them greet each other by using the the \"actor say\" block twice."
+        'studio_iceage_move_to_flag': "Scrat is looking for his acorn!  Help him go the last 200 pixels using the \"move\" block."
+        'studio_iceage_move_to_actor': "Diego (actor 1) is searching for Manny (actor 2). Help Diego get 200 pixels down to Manny using the \"move actor\" block. Next, connect a block to the \"when actor touches\" event to play a sound when they find each other."
+        'studio_iceage_repeat': "The \"repeat forever\" block allows you to run code continuously. Can you attach blocks to it to help Granny get some exercise by moving left and right repeatedly?"
+        'studio_iceage_click_hello': "Can you help Sid do a final call for friends when clicked? Don’t forget to click on him after you hit Run."
+        'studio_iceage_move_events': "Scrat is still looking for acorns! Help him scamper around by attaching the \"move\" block to each of the events and changing the direction to work for each arrow key. When you’re done, click Run and try to collect all of the acorns."
+        'studio_iceage_sound_and_points': "We’ve provided most of the code that keeps Granny moving left and right, as well as the code to move Sid around the screen. Can you finish the code and add blocks to the \"when actor touches\" event to make it so that you both play a sound and score a point when Sid and Granny collide? Press Run to play the game."
+        'studio_iceage_warn_ice_age': "Everyone needs to move faster because an ice age is coming! Change the background to \"icebergs\" and set Sid’s speed to fast, then press Run and move him to Diego to warn him."
+        'studio_iceage_throw_hearts': "Sid wants to let his grandmother, Granny, know that he loves her. Use the \"actor throw\" command to have Sid (actor 1) throw hearts at Granny (actor 2)! Set Granny's mood to happy when she gets hit with the hearts."
+        'studio_iceage_free_play': "Time to create your own game or story! When you're done, click Finish to let friends try your game on their phones, tablets, or computers!"
```

Comment:
some of these read pretty long to me, but i'm sure you've debated that separately.